### ISSUE TITLE
trained model prediction issue (negative or over 1 values)

### ISSUE THREAD

**Issue by dhlee-jubilee** [keras-team/autokeras#474]
I searched architecture and saved the model.
And then loaded the architecture and trained it as a general Keras training way such as,

trained_model = clf.fit(X_train, y_train, 
                              validation_data=(X_val, y_val), batch_size= batch_size, 
                              epochs = epochs, # epochs, verbose = 1, 
                              class_weight= class_weights, shuffle = True, callbacks=callbacks
                             )    

not like AutoKeras training way such as,

clf.final_fit(X_train, y_train, X_val, y_val, retrain=False, trainer_args={'max_iter_num':100})

Training was well, but the problem was inference step when the trained model predicts, such as,

y_pred = trained_model.predict(X_test, verbose=0)

the predicted values are sometimes negative values or over 1.
what's the problem??
